firmware/arduino/chessboard/motorDriver.ino

```diff
@@ -92,6 +92,16 @@ bool moveStraight(int motor[], float startCol, float startRow, float endCol, flo
   int dir, numSteps;
   int i;
 
+  //makes sure all arguments are in whole unitSpaces
```

Comment:
Need a space after the // and capitalize the 'm'

Reply:
Might want to reword this slightly, since whole and .5 may be confusing to someone reading the code. Maybe mention that  it makes sure all args can produce a multiple of unitspaces?